browser/brave_wallet/external_wallets_importer.cc

```diff
@@ -377,11 +377,21 @@ void ExternalWalletsImporter::GetMnemonic(bool is_legacy_crypto_wallets,
 
   std::unique_ptr<PasswordEncryptor> encryptor =
       PasswordEncryptor::DeriveKeyFromPasswordUsingPbkdf2(
-          password, *salt_decoded, 10000, 256);
+          password, *salt_decoded, 600000, 256);
   DCHECK(encryptor);
 
   auto decrypted_keyrings =
       encryptor->DecryptForImporter(*data_decoded, *iv_decoded);
+  if (!decrypted_keyrings) {
+    std::unique_ptr<PasswordEncryptor> encryptor_10k =
+        PasswordEncryptor::DeriveKeyFromPasswordUsingPbkdf2(
```

Comment:
Can we add a test for this logic as well?

Reply:
@kdenhartog we already have that test using `valid_data`. @supermassive maybe we should also rename that test data and also test suite name to indicate it's 10k iterations so it would be obvious

Reply:
Thanks for clarifying, I didn't see it in the limited view GH showed for changes and forgot to check already implemented code. This sounds like it's addressed so I'll resolve it.